kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -138,15 +139,17 @@ private PartitionedTableId getRecordTable(SinkRecord record) {
     TableId baseTableId = topicsToBaseTableIds.get(record.topic());
 
     PartitionedTableId.Builder builder = new PartitionedTableId.Builder(baseTableId);
-    if (useMessageTimeDatePartitioning) {
-      if (record.timestampType() == TimestampType.NO_TIMESTAMP_TYPE) {
-        throw new ConnectException(
-            "Message has no timestamp type, cannot use message timestamp to partition.");
-      }
-
-      builder.setDayPartition(record.timestamp());
-    } else {
-      builder.setDayPartitionForNow();
+    if(useIngestionTimePartitioning) {
+    	
+	  if (useMessageTimeDatePartitioning) {
+	    if (record.timestampType() == TimestampType.NO_TIMESTAMP_TYPE) {
+		  throw new ConnectException(
+	      "Message has no timestamp type, cannot use message timestamp to partition.");
+	    }
+	    builder.setDayPartition(record.timestamp());
+	  } else {
+        builder.setDayPartitionForNow();
```

Comment:
* Actually this is `IngestionTimePartitioning`
* invalid indentation

Reply:
Indentation is fixed now. Regarding the ingestion time partitioning, it's **not** as ingestion time partitioning doesn't require decorator. @yohei1126

Reply:
This is one of Ingestion Time Partitioning since it has two type: with partition decorator and without partition decorator. See the document. https://cloud.google.com/bigquery/docs/creating-partitioned-tables#creating_an_ingestion-time_partitioned_table_from_a_query_result

Reply:
Regardless of the partitioning strategy, this configuration/functionality to enable/disable partition decorator while inserting records into BigQuery. Hence the name.

Reply:
If so configuration name should be like usePartitionDecorator.

Reply:
It is `usePartitionDecorator` indeed.